byoc: update network related docs (#23353)
pingcap/docs: tidb-cloud/byoc/configure-aws-vpc-peering-access-cidr.md

summary: Learn how to configure allowed access CIDR rules for AWS VPC peering in TiDB Cloud BYOC.
---

# Configure Allowed Access CIDRs for AWS VPC Peering

This document describes how to configure allowed access CIDR rules for AWS VPC peering in TiDB Cloud BYOC.

AWS VPC peering creates a private connection between your VPC and TiDB Cloud, allowing your instance to be accessed privately without using the public internet. For new VPC peering requests, contact [TiDB Cloud Support](/tidb-cloud/tidb-cloud-support.md).

> **Note:**
>
> This document does not describe how to create an AWS VPC peering connection. VPC peering connections are created manually by you in your AWS account, or with assistance from the TiDB Cloud team. This document only describes how to configure which source CIDR ranges are allowed to access your instance through an existing VPC peering connection.

## How allowed access CIDRs work

For security reasons, all traffic from the peer VPC is denied by default, regardless of the VPC peering status. Even if a VPC peering connection is active, traffic can reach your TiDB Cloud BYOC instance only after you configure allowed access CIDR rules.

> Allowed access CIDR rules are applied at the instance level and are shared by all VPC peering connections for the instance. When you add or remove a CIDR rule, TiDB Cloud synchronizes the corresponding access rule to the security group of the private Network Load Balancer (NLB).

Reviewer: ```suggestion
Allowed access CIDR rules are applied at the instance level and shared across all VPC peering connections for the instance. When you add or remove a CIDR rule, TiDB Cloud synchronizes the corresponding access rule to the security group of the private Network Load Balancer (NLB).
```

The following behavior applies:

> - If no allowed access CIDR is configured, traffic through VPC peering is denied.

Reviewer: ```suggestion
- If no allowed access CIDRs are configured, all traffic through VPC peering is denied.
```

6. If you choose **Allow traffic from a specific IP range through VPC peering**, enter a CIDR range in the **Access CIDR** field, and then click **Add to List**.

    You can add multiple CIDR ranges. For example, you can add the CIDR ranges of multiple application subnets.

7. Review the **Access CIDRs List**, and then click **Update**.

After the configuration is updated, the allowed CIDR ranges are displayed in the **Allowed Access CIDR** area. If global access is enabled, TiDB Cloud displays a warning message indicating that traffic from all VPCs that can reach the private NLB through VPC peering is allowed.

## Recommendations

- Use specific application subnet CIDR ranges whenever possible.
- Avoid enabling global access for long-term production use unless your security requirements explicitly allow it.
- Keep the allowed access CIDR list aligned with your application subnet changes.
- If you have multiple VPC peering connections for the same instance, make sure the CIDR list covers all application subnets that need access.

> ## Troubleshooting

Reviewer: ```suggestion
## Troubleshooting

If connectivity is not working as expected after configuring allowed access CIDRs, review the following common issues and their solutions.
```
